config/routes.rb

```diff
@@ -687,6 +687,10 @@
           get   :add_favorite
           get   :remove_favorite
           get   :get_favorites
+          get   :get_standard_statements
+          get   :get_materials_standards
+          get   :add_materials_standard
```

Comment:
Should this and `remove_materials_standard` be a `post` instead for csrf protection?